src/components/routes.js

```diff
@@ -106,6 +107,12 @@ const supportPaths = [
     exact: false,
     linkText: 'Google Group',
     external: true
+  },
+  {
+    path: '/terms-of-service',
+    exact: true,
```

Comment:
What does exact mean? It's false everywhere else?

Reply:
good q. dunno. i changed it back to false. i had only changed it to true before b/c i was troubleshooting lol. the only component where exact == true is Introduction.

Reply:
from the react router docs: https://reacttraining.com/react-router/web/api/Route/exact-bool

> exact: bool
    When true, will only match if the path matches the location.pathname exactly.

path | location.pathname | exact | matches?
-- | -- | -- | --
/one | /one/two | true | no
/one | /one/two | false | yes

so.. I think it makes sense for `exact` to be `true` for introduction, which owns the `/` route, otherwise it would render in addition to every other route since `/` is a prefix for `/benchmarks` and so on. we don't have any nested routes, so we could probably set `exact` to `true` for every route.